src/Localization/Localization/src/ResourceManagerStringLocalizer.cs

```diff
@@ -221,5 +237,11 @@ private static partial class Log
     {
         [LoggerMessage(1, LogLevel.Debug, $"{nameof(ResourceManagerStringLocalizer)} searched for '{{Key}}' in '{{LocationSearched}}' with culture '{{Culture}}'.", EventName = "SearchedLocation")]
         public static partial void SearchedLocation(ILogger logger, string key, string locationSearched, CultureInfo culture);
+
+        [LoggerMessage(2, LogLevel.Debug, $"The resource with key '{{Key}}' is not found.", EventName = "ResourceNotFound")]
```

Comment:
```suggestion
        [LoggerMessage(2, LogLevel.Debug, $"A resource for '{{Key}}' with culture '{{Culture}}' was not found.", EventName = "ResourceNotFound")]
```

Reply:
I avoided this because the previous log already has the culture

Reply:
It's possible that the logs won't be right next to each other, and I don't think it's too costly to add one more parameter. I'd prefer we add it.